### ISSUE TITLE
Ensure that the Hardware Intrinsics feature is documented for 3.0

### ISSUE THREAD

**Issue by tannergooding** [dotnet/dotnet-api-docs#1672]
We should ensure that the hardware intrinsics feature, namely the types in the `System.Runtime.Intrinsics` namespace, is properly documented for 3.0

At minimum, this likely requires some cleanup of the existing documentation comments (which largely just reference the underlying instruction and the corresponding native intrinsic) and we may want to consider linking to the "Intel Intrinsics Guide" (similarly to what the MSVC Intrinsics documentation does).

**Comment by tannergooding** [comment]
CC. @danmosemsft, @CarolEidt, @fiigii

**Comment by fiigii** [comment]
> we may want to consider linking to the "Intel Intrinsics Guide" (similarly to what the MSVC Intrinsics documentation does).

Yes, that would be great. Shall we add the links in the XML comments of API source code? If yes, do you have an existing example?

**Comment by tannergooding** [comment]
I believe that the link needs to be an `aka.ms` link so we can have it redirect if the URL ever changes. Otherwise, I believe we can just add it as needed.

**Comment by fiigii** [comment]
> I believe that the link needs to be an aka.ms link so we can have it redirect if the URL ever changes. 

I have contacted the team that is responsible for Intel Intrinsics Guide, and they said the URLs won't be changed in the future.

**Comment by ganchito55** [comment]
IMHO at least a link to https://software.intel.com/sites/landingpage/IntrinsicsGuide/ should be added at the end/beginning of the documentation. However it would be good if each instruction had each own link.

**Comment by tannergooding** [comment]
This is being worked on and I hope to have this resolved relatively soon.

**Comment by ganchito55** [comment]
I have created an example of usage with a benchmark (SIMD vs classic implementation vs LINQ implementation) that maybe could help you to improve the documentation.
https://github.com/ganchito55/simd
